Supervisor panic on shutdown

### Bug description

```
panic: sync: negative WaitGroup counter

goroutine 32 [running]:
sync.(*WaitGroup).Add(0xc0005a0050, 0xffffffffffffffff)
	sync/waitgroup.go:74 +0x147
sync.(*WaitGroup).Done(0xc0005a0050)
	sync/waitgroup.go:99 +0x34
github.com/gitpod-io/gitpod/supervisor/pkg/supervisor.socketActivationForDocker(0xf70648, 0xc0003c1240, 0xc0005a0050, 0xc0003c13c0)
	github.com/gitpod-io/gitpod/supervisor/pkg/supervisor/supervisor.go:880 +0x465
created by github.com/gitpod-io/gitpod/supervisor/pkg/supervisor.Run
	github.com/gitpod-io/gitpod/supervisor/pkg/supervisor/supervisor.go:200 +0x1725


```

### Steps to reproduce

Start a workspace and shutdown

### Expected behavior

No panic

### Example repository

-

### Anything else?

-

--- a/components/supervisor/pkg/supervisor/supervisor.go
+++ b/components/supervisor/pkg/supervisor/supervisor.go
@@ -197,6 +197,7 @@ func Run(options ...RunOption) {
 	go startSSHServer(ctx, cfg, &wg)
 	wg.Add(1)
 	go taskManager.Run(ctx, &wg)
+	wg.Add(1)
 	go socketActivationForDocker(ctx, &wg, termMux)
 
 	if !cfg.isHeadless() {

